cyclops-ui/src/components/pages/Modules/Modules.tsx

```diff
@@ -64,6 +87,16 @@ const Modules = () => {
     };
   }, []);
 
+  const populateNamespaceData = (allData: ModuleInterface[]) => {
+    const namespaceData = allData
+      .map((module) => module.targetNamespace)
+      .filter((targetNamespace, index, self) => {
+        return self.indexOf(targetNamespace) === index;
+      });
+    setNamespaceFilterData(namespaceData);
+    setModuleNamespaceFilter(namespaceData);
```

Comment:
We cannot use `setModuleNamespaceFilter` here because it is called every 10 seconds and resets the filter.

Reply:
true didnt check this my bad, will update